gradle/documentation/render-javadoc.gradle

```diff
@@ -118,6 +118,15 @@ configure(project(":solr:modules:analysis-extras")) {
   }
 }
 
+configure(project(":solr:modules:jwt-auth")) {
+  project.tasks.withType(RenderJavadocTask) {
+    // TODO: clean up split packages
+    javadocMissingIgnore = [
+        "org.apache.solr.security"
```

Comment:
Since you moved everything to o.a.s.security.jwt, do we still need this?

Reply:
You're rifht, removing